packages/orbit-components/src/Popover/types.d.ts

```diff
@@ -34,4 +34,5 @@ export interface Props extends Common.Globals {
   readonly zIndex?: number;
   readonly ariaLabel?: string;
   readonly ariaLabelledby?: string;
+  readonly role?: "dialog" | "menu" | "grid" | "listbox" | "tree";
```

Comment:
what do you think about something like this?

```
export type PopoverRole = "dialog" | "menu" | "grid" | "listbox" | "tree";
```

and then:
```
readonly role?: PopoverRole;
```

and import it in `ContentWrapper.tsx` as well?